index.html

```diff
@@ -4,12 +4,26 @@
     <link rel="stylesheet" href="css/style.css">
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Type-Racer</title>
 </head>
 <body>
+    <div class="main">
+        <div class="content">
+            <div class="typing-race">
+                <div class="text-for-typing">
```

Comment:
if this div will remain empty, you can write it in one line, but it's just a code style tip